### ISSUE TITLE
Constant scoping

### ISSUE THREAD

**Issue by v0dro** [pgtgrly/GRruby-extension#12]
https://github.com/pgtgrly/GRruby-extension/blob/56240dadb14aa978faf67782933b8bb8edbb1f64/lib/grruby/tasks/gr_function_classes.rb#L293

Ideally constants like these should be placed inside a separate top-level name like `Rubyplot::COLOR_INDEX` for accessibility throughout the gem.

See how daru organises these: https://github.com/SciRuby/daru/blob/master/lib/daru.rb#L8
